src/Components/RecsListTable/RecsListTable.cy.js

```diff
@@ -661,13 +671,33 @@ describe('successful non-empty recommendations list table', () => {
     cy.get(ROWS_TOGGLER).click();
     cy.get(TABLE)
       .find('.pf-c-table__expandable-row.pf-m-expanded')
-      .each((el) => {
+      .each((el, index) => {
         // contains description
         cy.wrap(el).contains(
           'Sed ut perspiciatis unde omnis iste natus error.'
         );
         // contain total risk label
         cy.wrap(el).find('.ins-c-rule-details__stack');
+        // contains link to the clusters
+        const recommendationData = filterData()[index];
+        cy.wrap(el)
+          .find('.ins-c-rule-details__view-affected')
+          .find('a')
```

Comment:
Perhaps, we could use
```suggestion
          .find('.ins-c-rule-details__view-affected a')
```
?